boards/common/nrf52xxxdk/include/gpio_params.h

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @ingroup     boards_common_nrf52
+ * @ingroup     boards_common_nrf52xxxdk
```

Comment:
Doxygen group defined here:

https://github.com/RIOT-OS/RIOT/blob/0afed1de2023d1aab33ce01dc4427b4b5fbbde3d/boards/common/nrf52xxxdk/include/periph_conf_common.h#L9-L19

So it was indeed missorted before